examples/04_fuzzy_joining.py

```diff
@@ -156,44 +171,31 @@
 # Let's do some more inspection of the merging done.
```

Comment:
You also have on L165 a ``#noqa`` appearing that should be removed (see example rendering)